ccmlib/common.py

```diff
@@ -366,8 +366,14 @@ def isDse(install_dir):
 
 
 def isScylla(install_dir):
+    scylla_version = os.environ.get('SCYLLA_VERSION', None)
```

Comment:
1. Is this python3 related fix?  Seems orthogonal and so should be done in a separate patch.

2. why do it unconditionally?
can't it be done only `if install_dir is None:`

Reply:
as for 1, yes it can be separated into it's own commit

Reply:
as for 2, yes we could put it into the right context

Reply:
opened
https://github.com/scylladb/scylla-ccm/pull/189
for this one